modules/vpc/output.tf

```diff
@@ -0,0 +1,9 @@
+output "public_subnet_id" {
+  description = "Public Subnet ID"
+  value = aws_subnet.Public.*.id
```

Comment:
You need to run `terraform fmt --recursive` in order to align style

Reply:
fixed